### ISSUE TITLE
Workshop and other events with precon deck should try to use the display name from MTGA

### ISSUE THREAD

**Issue by Porkepix** [Manuel-777/MTG-Arena-Tool#1075]
It currently use some technical name coming from internals of the game, as seen on this screenshot:
![history_event_text](https://user-images.githubusercontent.com/709500/82267982-284feb80-996e-11ea-8ec6-ef2298fdd847.PNG)
If logs allow it, it would be nicer to grab the name displayed within the game.

**Comment by Manuel-777** [comment]
Under `C:\Program Files (x86)\Wizards of the Coast\MTGA\MTGA_Data\Downloads\Loc\loc_Decks_%HASH%.mtga` we can find the translations of these now.

**Comment by Porkepix** [comment]
Oh, good to know! Then I guess that also applies to name of the events, for the Economy and History tabs?

**Comment by Manuel-777** [comment]
Yes, they are under the `loc_Events` file, their names are filed as `Events/Event_Cat_Event_CodeName` it seems, I will see if I can feed the database with it
